test/test_utils/__init__.py

```diff
@@ -52,10 +52,10 @@ def get_ami_id_ssm(region_name, parameter_path):
     ami_id = eval(ami['Parameter']['Value'])['image_id']
     return ami_id
 
-UBUNTU_18_BASE_DLAMI_US_WEST_2 = "ami-067ab1a84ae92e377" # get_ami_id_boto3(region_name="us-west-2", ami_name_pattern="Deep Learning AMI GPU CUDA 11.1.1 (Ubuntu 18.04) ????????")
-UBUNTU_18_BASE_DLAMI_US_EAST_1 = "ami-08f77506502c3af58" # get_ami_id_boto3(region_name="us-east-1", ami_name_pattern="Deep Learning AMI GPU CUDA 11.1.1 (Ubuntu 18.04) ????????")
-AML2_GPU_DLAMI_US_WEST_2 = "ami-05f901a03eb17318a" # get_ami_id_boto3(region_name="us-west-2", ami_name_pattern="Deep Learning AMI GPU CUDA 11.1.1 (Amazon Linux 2) ????????")
-AML2_GPU_DLAMI_US_EAST_1 = "ami-0acceec125660ec66" # get_ami_id_boto3(region_name="us-east-1", ami_name_pattern="Deep Learning AMI GPU CUDA 11.1.1 (Amazon Linux 2) ????????")
+UBUNTU_18_BASE_DLAMI_US_WEST_2 = get_ami_id_boto3(region_name="us-west-2", ami_name_pattern="Deep Learning Base AMI (Ubuntu 18.04) Version 55.0")
```

Comment:
Please change ami_name_pattern to "Deep Learning Base AMI (Ubuntu 18.04) Version ??.?" to pick the latest Base AMI. Apply similar changes to the other AMIs.

Reply:
@RadhikaB-97 Why was CUDA ami used in the first place?